scripts/artifact-building/amphora-image/amphora-image-push-to-mirror.yml

```diff
@@ -0,0 +1,46 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This playbook's function is to upload the git clones for all
+# git repositories use by RPC-O to rpc-repo.
+
+- name: Push the git artifacts to rpc-repo
+  hosts: mirrors
+  vars:
+    working_dir: "{{ lookup('ENV', 'PWD') }}"
+    src_amp_archive_dir: "{{ working_dir }}/amp-image/{{ rpc_release }}"
+    dst_amp_archive_dir: "/var/www/repo/amp-image"
```

Comment:
Can we make this ``/var/www/repo/images/amphora`` ? I expect we'll be doing more images in the future and would prefer them to be in a collective bucket.